test/src/main/java/org/apache/accumulo/test/functional/DurabilityIT.java

```diff
@@ -218,7 +192,7 @@ private void restartTServer() throws Exception {
     cluster.start();
   }
 
-  private long writeSome(AccumuloClient c, String table, long count) throws Exception {
+  private void writeSome(AccumuloClient c, String table, long count) throws Exception {
     int iterations = 5;
```

Comment:
@Manno15 could `iterations`, `attempts`, and the for loop be removed?  Seems like this loop only existed to write multiple times and get the median time.

Reply:
Yeah, that appears to be the case. I'll remove it and then test it. Thanks